src/browser-frontend/views/browser/chrome/tabbar/tab/tab-favicon.jsx

```diff
@@ -36,15 +35,28 @@ export default class TabFavicon extends PureComponent {
 
     if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.INITIAL ||
         this.props.loadState === DomainPageMetaModel.LOAD_STATES.CONNECTING) {
-      contents = <SpinnerType1 />;
+      contents = (<Spinner
+        src="var(--theme-tab-connecting)"
+        width="28px"
+        height="16px"
+        mode="contain"
+        className="spinner"
```

Comment:
hmm, I suppose we might want to spin it differently. b ut you're right, I can just add a class for that. Will change it.